publication-commons/src/main/java/no/unit/nva/publication/permissions/publication/PublicationStrategyBase.java

```diff
@@ -51,6 +51,25 @@ protected boolean userRelatesToPublicationThroughPublicationOwnerOrCuratingInsti
         return userIsFromSameInstitutionAsPublicationOwner() || userBelongsToCuratingInstitution();
     }
 
+    protected boolean userRelatesToPublicationThroughPublicationOwnerOrCuratingInstitutionOrChannelClaim() {
+        return userRelatesToPublicationThroughPublicationOwnerOrCuratingInstitution() || userBelongsToPublicationChannelOwner();
+    }
+
+    private boolean userBelongsToPublicationChannelOwner() {
+        if (isNull(userInstance)) {
+            return false;
+        }
+
+        var claimedPublicationChannel = resource.getPrioritizedClaimedPublicationChannel();
+
+        if (claimedPublicationChannel.isEmpty()) {
+            return false;
+        }
+
+        var channelOwner = claimedPublicationChannel.get().getCustomerId();
+        return userInstance.getCustomerId().equals(channelOwner);
```

Comment:
We check that userInstance is not null but we do not check that userInstance.customerId is null